docs/data/material/customization/containerQueries/containerQueries.md

```diff
@@ -0,0 +1,91 @@
+# Container Queries
+
+<p class="description">API that enables the use of CSS container queries based on theme breakpoints.</p>
+
+A utility function based on the [theme breakpoints](/material-ui/customization/breakpoints/) for creating [CSS container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries).
+
+## Usage
+
+To create a container query, use `theme.containerQueries` with any method available in the [theme.breakpoints](/material-ui/customization/breakpoints/#api).
+The value can be a unitless (pixel), a string, or a breakpoint key, for example:
+
+```js
+theme.containerQueries.up('sm'); // => '@container (min-width: 600px)'
+```
+
+:::info
+One of the ancestors must have the CSS container type specified.
+:::
+
+{{"demo": "BasicContainerQueries.js"}}
+
+### Named containment contexts
+
+To refer to a [containment context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries#naming_containment_contexts), call the `containerQueries` method with the name, then you will be able to access all the breakpoint methods like the example above:
```

Comment:
```suggestion
To refer to a [containment context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries#naming_containment_contexts), call the `containerQueries` method with the container name. 
That's how you can access all the breakpoint methods like the example above:
```

I'm assuming "the name" refers to the container name? And a tiny tweak here to remove the future tense.